test/unit/compiler/passes/report-incorrect-plucking.spec.js

```diff
@@ -0,0 +1,37 @@
+"use strict";
+
+const chai = require("chai");
+const helpers = require("./helpers");
+const pass = require("../../../../lib/compiler/passes/report-incorrect-plucking");
+
+chai.use(helpers);
+
+const expect = chai.expect;
+
+describe("compiler pass |reportIncorrectPlucking|", function() {
+  it("prevents \"@\" from being used with an action block", function() {
+    expect(pass).to.reportError("start1 = 'a' @'b' 'c' { /* empty action block */ }", {
+      message: "\"@\" cannot be used with an action block at line 1, column 14."
```

Comment:
```suggestion
      message: "\"@\" cannot be used with an action block at line 1, column 14.",
      location: {
        start: { offset: 13, line: 1, column: 14 },
        end: { offset: 17, line: 1, column: 18 }
      }
```
Actually, text in the message should point to the beginning of the action block (opening `{`), but right now:
- that information is unavailable (action block `location` covers expression and the action itself)
- other check passes that work with actions also should be modified

So for now will live with that